common/events/player_event_logs.cpp

```diff
@@ -707,8 +1265,96 @@ void PlayerEventLogs::SetSettingsDefaults()
 	m_settings[PlayerEvent::PARCEL_RETRIEVE].event_enabled           = 1;
 	m_settings[PlayerEvent::PARCEL_DELETE].event_enabled             = 1;
 	m_settings[PlayerEvent::BARTER_TRANSACTION].event_enabled        = 1;
+	m_settings[PlayerEvent::SPEECH].event_enabled                    = 1;
 
 	for (int i = PlayerEvent::GM_COMMAND; i != PlayerEvent::MAX; i++) {
 		m_settings[i].retention_days = RETENTION_DAYS_DEFAULT;
 	}
 }
+
+void PlayerEventLogs::LoadEtlIds()
+{
+	m_etl_settings.clear();
+	m_etl_settings = {
```

Comment:
Action item we need to make a world CLI command that shows these settings in JSON so Spire can consume what the source says is defined and active.

Reply:
Added a world cli command.